packages/kit/src/runtime/client/client.js

```diff
@@ -1354,6 +1354,7 @@ export function create_client(app, target) {
 			return invalidate();
 		},
 
+		// TODO: Why is only this camel cased?
```

Comment:
No particular good reason, as far as I can tell, since some other things from here that are exposed in `packages/kit/src/runtime/app/navigation.js` are snake case in this file and then camel case in that file.

Reply:
This was added in #6493. I assume it was just a tiny slip-up. I'd be in favor of switching to `invalidate_all`.